scripts/test.sh

```
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+set -e
+
+# Test Runner Script
+# Runs all tests in the Circles solution
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PROJECT_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+SOLUTION_FILE="$PROJECT_ROOT/Circles.sln"
```

Shellcheck (0.11.0)

[warning] SOLUTION_FILE appears unused. Verify use (or export if used externally).

(SC2034)